osr2mp4/osrparse/replay.py

```diff
@@ -14,74 +14,83 @@ def __init__(self, time_since_previous_action, x, y, keys_pressed):
 
 
 class Replay(object):
-	__BYTE = 1
-	__SHORT = 2
-	__INT = 4
-	__LONG = 8
+	"""
+		R.A.P.E
+
+		R - redz
+		A - amazing
+		P - parser for replay
+		E - epic
+	"""
+
+	game_mode: int = 0
+	game_version: int = 0
+	beatmap_hash: str = ''
+	player_name: str = ''
+	replay_hash: str = ''
+	number_300s: int = 0
+	number_100s: int = 0
+	number_50s: int = 0
+	gekis: int = 0
+	katus: int = 0
+	misses: int = 0
+	score: int = 0
+	max_combo: int = 0
+	is_perfect_combo: int = 0
+	mod_combination: [Mod] = []
+	life_bar_graph: str = ''
+	timestamp: datetime.datetime = None
+	play_data: [ReplayEvent] = []
+
+	# internal
+	view: [memoryview, bytes] = None # memoryview or bytes
```

Comment:
`Union[memoryview, bytes]`?

Reply:
havent tried the typing stuff yet, will change this